agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AiContextKeys.java

```diff
@@ -13,5 +13,8 @@ public final class AiContextKeys {
   public static final ContextKey<String> ROLE_NAME =
       ContextKey.named("applicationinsights.internal.role_name");
 
+  public static final ContextKey<String> USER_AGENT =
+      ContextKey.named("applicationinsights.internal.user_agent");
+
```

Comment:
wondering why we need this? upstream was just a rename of existing attribute, but this seems new?

Reply:
attributes do not have user_agent anymore. I can check main branch what was it before. this is used temporarily and will get ignored in the exporter.

Reply:
i checked.. `http.user_agent` is part of span attributes and `user_agent.original` is not.

Reply:
Here is the diff:

**old attributes:** 
http.flavor value: 1.1
http.method value: GET
http.route value: /HttpPreaggregatedMetrics/*
http.scheme value: http
http.status_code value: 200
http.user_agent value: AlwaysOn
net.host.name value: localhost
net.host.port value: 55190

**new attributes:**
http.method value: GET
http.route value: /HttpPreaggregatedMetrics/
http.scheme value: http
http.status_code value: 200
net.host.name value: localhost
net.host.port value: 55405

Reply:
`http.flavor` was split into `net.transport.name` and `net.transport.version` (https://github.com/open-telemetry/opentelemetry-specification/pull/3272)
`http.user_agent` was renamed to `user_agent.original` (https://github.com/open-telemetry/opentelemetry-specification/pull/3190)

are these new attributes not showing up for some reason?